FreeRTOS/FreeRTOS
# Empty project, base project for WinArv is wrong

Exemplary project for Arduino Uno board with qemu-system-avr support.
-----------

This project bases on AVR_ATMega323_WinAVR example. The MCU and its port was changed. Make scripts ware extended to add support for qemu and visual code editor.

Test Steps
-----------
Go to FreeRTOS/Demo/AVR_ATMega328_ArduinoUno
make
make gdbserver
attach to gdbserver

Checklist:
----------
I don't expect any surprises. I just added in in order to avoid braking many times opened doors.

Related Issue
-----------


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
